docs/Tutorials/Orientation.md

```diff
@@ -2,50 +2,60 @@
-among them. To assign a Global CNS to the Domain, one may arbitrarily assign
-corner ids to each of the twelve corners in the Domain, so long as each corner
-has a single, unique id.
-
-\note This Global CNS assumes that there is no additional identifying of faces
-with one another for periodic boundary conditions. That is, that no block has
-itself as a neighbor. If you wish to additionally identify faces of the same
-block with each other, that must be done in an additional step. This step is
-explained in the "Setting Periodic Boundary Conditions" section.
-
-### Local Corner Numbering Scheme (Local CNS)
-The orientation of the logical axes in each Block define an ordered set of
-corners. Once the logical axes in each Block are determined, it is possible
-to write down a Local CNS:
-
-Each Block has \f$2^{dim}\f$ local corners, numbered from \f$0\f$ to
-\f$2^{dim} - 1\f$. We will take the three-dimensional cube as an example:
-This Block has 8 local corners (regardless how many of these are
-shared), numbered from 0 to 7. The corners are labelled according to the
-CoordinateMap corresponding to the Block, as follows:
+# OrientationMap {#tutorial_orientations}
+
+### Introduction
+Each element in a domain has a set of internal directions which it uses
+for computations in its own local coordinate system. These are referred to
+as the logical directions \f$\xi\f$, \f$\eta\f$, and \f$\zeta\f$, where
+\f$\xi\f$ is the first dimension, \f$\eta\f$ is the second dimension, and
+\f$\zeta\f$ is the third dimension. In a
+domain with multiple elements, the logical directions are not necessarily
+aligned on the interfaces between two elements, as shown in the figure below.
+As fluxes need to be communicated across the boundaries of adjacent elements,
+there needs to be a system in place that organizes the relative orientations of
+elements with respect to their neighbors. This is handled by OrientationMap.
+
+### %OrientationMaps between %Blocks
+Each Block in a Domain has a set of BlockNeighbors, which each hold an
+OrientationMap. In this scenario, the Block is referred to as the host, and
+the OrientationMap held by each BlockNeighbor is referred to as "the
+orientation the host Block has with respect to this BlockNeighbor." This is
+a convention, so we give an example of constructing and assigning the correct
+OrientationMaps:
+
+\image html OrientationMap.png "Two neighboring blocks."
+
+In the image above, we see a domain decomposition into two Blocks, which have
+their logical axes rotated relative to one another. With the left block as
+the host Block, we see that it has a BlockNeighbor in the \f$+\xi\f$ direction.
+The OrientationMap held by the BlockNeighbor determines the mapping from each
+logical direction in the host Block to that in the neighoring Block. That is,
+the OrientationMap takes as input local information and returns neighbor
+information. In this case, the \f$+\xi\f$, \f$+\eta\f$, \f$+\zeta\f$ directions
```

Comment:
I am still confused by the wording.  How about making it even more explicit: "...takes as input local information (i.e. coordinates in the host's coordinate system) and returns neighbor information (i.e. coordinates in the neighbor's coordinate system)".  I think the reason that this is confusing is that 'local' refers to the host, but since the OrientationMap is owned by the neighbor one may think that 'local' refers to the neighbor.

Reply:
I think a confusing thing is that the host Block holds an object named BlockNeighbor, which itself holds an OrientationMap which I am calling the orientation of the host with respect to the neighbor. The neighboring Block itself holds a NeighborBlock which represents the host block. I will try to clear this up more.